flink-clients/src/test/java/org/apache/flink/client/RemoteExecutorHostnameResolutionTest.java

```diff
@@ -88,15 +65,14 @@ private static void checkPreconditions() {
 		// the test can only work if the invalid URL cannot be resolves
 		// some internet providers resolve unresolvable URLs to navigational aid servers,
 		// voiding this test.
-		boolean throwsException;
 		try {
 			//noinspection ResultOfMethodCallIgnored
 			InetAddress.getByName(nonExistingHostname);
-			throwsException = false;
 		}
 		catch (UnknownHostException e) {
-			throwsException = true;
+			return;
 		}
-		assumeTrue(throwsException);
+
+		fail("Expected UnknownHostException but none was thrown.");
```

Comment:
We could put this inside the `try` block. Then we would get rid of this lonely `return` statement.